.github/workflows/install-shared-dependencies/action.yml

```diff
@@ -48,12 +49,24 @@ runs:
           run: |
               yum install -y gcc pkgconfig openssl openssl-devel which curl redis6 gettext --allowerasing
 
+        - name: Install software dependencies for Ubuntu MUSL
+          shell: sh
+          if: "${{ inputs.os == 'ubuntu-latest-musl' }}"
```

Comment:
is it supposed for node only or for all clients?
if for node only, move this to node CI and revert this file

Reply:
We should support for all languages.
We should take this a step forward and test the package on all languages alpine versions.
Theres no dependency between them tho, we can provide support first to one language alpine, and in a later point to provide support to other languages. 
But since we already know that we will provide supports to all in the future I think its ok to lay the foundation for the others. 
Its need to be well documented so the developer who will build the support for other languages will have an easy life understanding the foundations and how he can use it.

Reply:
Cosider updating DELEVOPER.md

Reply:
It's not nodejs specific, it installs shared glide dependencies for musl, it should be located here

Reply:
This part is shared and needed for all `musl` / `alpine`,   as @barshaul  said.  Not Node specific.
I agree in general it should support all languages,   and I strongly agree with @avifenesh to support at this stage only one lang (Node) and expand later.